src/kiota/Handlers/Plugin/EditHandler.cs

```diff
@@ -62,7 +62,10 @@ public required Option<string> PluginAuthRefIdOption
         get; init;
     }
 
-
+    public required Option<bool> NoWorkspaceOption
```

Comment:
I thought we said we wouldn't add this option for any other command than add?

Reply:
Edit was also feasible to avoid workspace management

Reply:
I don't understand how this is supposed to work since edit is supposed to read all arguments that are not passed by the use from the workspace itself? so we'll loose some of the information in the process?

Reply:
Changes reverted in edit handler, I saw that edit command supports to pass parameters but I have reverted changes for alignment with workspace usage